install.sh

```diff
@@ -81,17 +81,32 @@ sudo ln -s `pwd`/crew $CREW_PREFIX/bin
 mkdir $CREW_LIB_PATH/lib && cd $CREW_LIB_PATH/lib
 wget -N -c $URL/lib/package.rb
 wget -N -c $URL/lib/package_helpers.rb
-#create the device.json file
+
+#create or update the device.json file
+ruby_version='2.0.0p247'
```

Comment:
The point here is we don't install actuall ruby package.

For example, we can make minimum bootstrap ruby and install it with a version number like "bootstrap".  Then, we can update ruby.rb to 2.4.0 safely.  @thedamian gave me this enlighten idea in #340.

Therefore, I want to hard code the version number in somewhere of install.sh.